GoogleMaps_HUD/gmaps_hud/src/main/java/sky4s/garminhud/app/NotificationMonitor.java

```diff
@@ -120,8 +120,7 @@ private static void logi(String msg) {
         @Override
         public void onReceive(Context context, Intent intent) {
 
-            int showETAValue = intent.getIntExtra(Integer.toString(R.id.switchShowETA), 0);
-            showETA = 0 == showETAValue ? showETA : 2 == showETAValue;
+            showETA = intent.getBooleanExtra(Integer.toString(R.id.switchShowETA), false);
```

Comment:
I think there will be a little dangerous.
When I switch showETA in MainAcitivity, it'ok.
When I switch other option in MainAcitivity (but there is no other option right now), not switchShowETA extra in intent, showETA will be turn off.

So I change from send bool to send integer, for two or more options in MainAcitivity.

Reply:
Hm, got what you mean.

Just a thought, but may it is possible too, to use the follwing code:
showETA = intent.getBooleanExtra(Integer.toString(R.id.switchShowETA), showETA);

Reply:
It should be ok!